### DISCUSSION TITLE
Additional logging while initialization

### DISCUSSION THREAD

**Discussion by SGFGOV** [medusajs/medusa#5087] [Feature Requests]
When you have several plugins,
it will be good to know which plugins are initialised correctly, it also gives some confidence, that there are no accidental infinite loops in plugin initialisations. 
Govind

**Comment by tsvetann** [comment]
I like this idea

**Comment by tsvetann** [comment]
@SGFGOV we should probably come up with PR for this?

**Reply by SGFGOV** [reply]
Not sure if its worth It's not really worth a pr coz it's literally a 1 line statement:)..I am hoping the core team will just edit one line :)

**Reply by tsvetann** [reply]
it can be a config. enable_debug = true

**Reply by SGFGOV** [reply]
@tsvetann I have tried a simple implementation..care to review plz?
